app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center,
+                 clean_power,
+                 average_rating,
+                 count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car):
+        price = self.clean_power - car.clean_mark
+        price *= car.comfort_class
+        price *= self.average_rating / self.distance_from_city_center
+        return round(price, 1)
+
+    def wash_single_car(self, car):
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, rate):
+        total_rate = self.average_rating * self.count_of_ratings
+        total_rate += rate
```

Comment:
` total_rate = self.average_rating *
                      self.count_of_ratings +
                      rate`
I think this will be better )

Reply:
Yep, thank you:)